src/main/java/sign_up/interface_adapters/SignUpViewModel.java

```diff
@@ -0,0 +1,35 @@
+package sign_up.interface_adapters;
+
+import sign_up.use_case.*;
+
+public class SignUpViewModel implements SignUpOutputBoundary {
+    // The view which will be updated following the viewmodel
+    private final View view;
+    private final SignUpInputBoundary inputBoundary;
+
+    public SignUpViewModel(SignUpDataAccessInterface dataAccess, View view) {
```

Comment:
Can we use the "Gateway" suffix for the data access to keep things consistent?

Reply:
For sure I will update that